.github/workflows/ci.yml

```diff
@@ -46,12 +46,10 @@ jobs:
         uses: subosito/flutter-action@v2
         with:
           cache: true
-      - name: Set file permissions
-        run: chmod +x tools/flutter-dependencies.sh
       - name: Install dependencies
         run: |
-          echo "publish_to: none" >> packages/flutter/pubspec.yaml
-          tools/flutter-dependencies.sh
```

Comment:
Should the flutter-dependencies.sh file be deleted from the tools folder?

Reply:
I’ve kept it in case it serves any purpose when developing locally.

Reply:
Should I remove the 2 script files?

Reply:
No, it is not forced to delete.
But the developer can also run the code inside the script manually

Reply:
Alright so let's keep it for now